thredded.gemspec

```diff
@@ -22,9 +23,9 @@ Gem::Specification.new do |s|
   s.add_dependency 'htmlentities'
   s.add_dependency 'kaminari'
   s.add_dependency 'nokogiri'
-  s.add_dependency 'q'
   s.add_dependency 'rails', '>= 4.0.0'
   s.add_dependency 'rb-gravatar'
+  s.add_dependency 'activejob_backport' if Thredded::RailsVersion.less_than_4_2?
```

Comment:
Yeah, it doesn't work like this, gem dependencies are determined at the time of building the gem